.github/actions/oblt-cli-create-custom/action.yml

```diff
@@ -72,6 +75,8 @@ runs:
       run: |
         {
           echo -n '--template=${{ inputs.template }} '
+          [ -n '${{ inputs.cluster-name }}' ] \
```

Comment:
Possible injection

Reply:
I'll be raising a follow-up to avoid the injection, rather than changing here the `cluster-name` 👼